std/js/Browser.hx

```diff
@@ -28,6 +28,9 @@ import js.html.XMLHttpRequest;
 class Browser {
 	/** The global window object. */
 	public static var window(get, never):js.html.Window;
+	
+        /** The global scope with additional fields available only in a worker context. */
+	public static var self(get, never):js.html.DedicatedWorkerGlobalScope;
```

Comment:
But where's `get_self`? :)

Reply:
:P good catch, thanks @nadako, will close and re-open fixed tomorrow